sample_client/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
All these  `sample_client/.idea` files are pointless and are just noise and will receive too much changes. 
It should be handled by

https://github.com/owncloud/android-library/blob/master/.gitignore#L5-L6

I guess your root cause is you didn't open project from root directory

Reply:
good point. Lets .gitignore them.